### PR TITLE
Coverlet collector updates

### PR DESCRIPTION
**Pull request by ccaprio97**
## What’s changing?

To update dependencies, ran the 5 steps on the following link:
https://github.com/18F/piipan/blob/dev/docs/update-deps.md#semi-automated-steps 

on the following directories:
https://github.com/18F/piipan/pulls?q=is%3Apr+is%3Aopen+coverlet.collector 

## Why?

Fix these dependabot findings:
https://github.com/18F/piipan/pulls?q=is%3Apr+is%3Aopen+coverlet.collector

## This PR has:

-All automated updates from running the 5 step's commands
-A space in the index was added in order to commit again after having trouble pushing to github due to git bash GFE issues

##Extra Notes

Adding Bill and Michael as reviewers. With this being first time PR I wouldn't be surprised if something is wrong. Please feel free to let me know if anything needs to be changed or fixed

### CODE REVIEW

File: query-tool/src/Piipan.QueryTool/Pages/Index.cshtml [hunk 1]
```diff
@@ -44,7 +44,7 @@
             <legend class="usa-sr-only">participant information</legend>
             <label class="usa-label" asp-for="Query.FirstName"></label>
             <input class="usa-input" type="text" asp-for="Query.FirstName" />
-            <label class="usa-label" for="Query_MiddleName">@Html.DisplayNameFor(m => m.Query.MiddleName) <span class="text-gray-50">(optional)</span></label>
+            <label class="usa-label" for="Query_MiddleName">@Html.DisplayNameFor(m => m.Query.MiddleName) <span class="text-gray-50">(optional) </span></label>
```
[the remark this answers is not in the document]
Comment (ccaprio97) [reply query-tool/src/Piipan.QueryTool/Pages/Index.cshtml]: Fixed

File: participants/tests/Piipan.Participants.Core.IntegrationTests/Piipan.Participants.Core.IntegrationTests.csproj [hunk 2]
```diff
@@ -9,15 +9,15 @@
   <ItemGroup>
     <PackageReference Include="Dapper" Version="2.0.123" />
     <PackageReference Include="Npgsql" Version="6.0.3" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.22" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0" />
```
[the remark this answers is not in the document]
Comment (ccaprio97) [reply participants/tests/Piipan.Participants.Core.IntegrationTests/Piipan.Participants.Core.IntegrationTests.csproj]: Fixed

**Review by kingcomma — APPROVED** [review]
👍 LGTM!

**Comment by kingcomma** [comment]
@ccaprio97 Now that this PR has one approval, feel free to merge it in.

**Pull request merged**